test/wallet-test.js

```diff
@@ -1301,37 +1303,49 @@ describe('Wallet', function() {
     importedKey = key;
   });
 
+  it('should require account key to create watch only wallet', async () => {
+    try {
+      watchWallet = await wdb.create({
+        watchOnly: true
+      });
+    } catch (e) {
+      assert.strictEqual(
+        e.message,
+        'Must add HD public keys to watch only wallet.'
+      );
+    }
+
+    const privateKey = PrivateKey.generate();
+    const xpub = privateKey.xpubkey('main');
+    watchWallet = await wdb.create({
+      watchOnly: true,
+      accountKey: xpub
+    });
+  });
+
   it('should import pubkey', async () => {
     const key = KeyRing.generate();
     const pub = new KeyRing(key.publicKey);
 
-    const wallet = await wdb.create({
```

Comment:
Why don't leave these test cases independent ? (I mean watchWallet)

Reply:
The change in the PR would fail these two tests. I could leave them as is, but would have to add the newly-required `accountKey` option to the `wdb.create`'s in each test. I figured the more elegant layout is to test wallet creation _once_, then test importing key and importing address in separate tests. Do you think I should re-write it the other way?

Reply:
For most cases, when possible, best to have the tests independent. The exception would be if there is significant setup, which I don't think there is here.

Reply:
Resolved at https://github.com/bcoin-org/bcoin/pull/639/commits/43d9ffadeffadfaa39b3df87beb800103f5c1ed5